package.json

```diff
@@ -38,6 +38,7 @@
         "@typescript-eslint/parser": "^3.0.1",
         "babel-loader": "^8.1.0",
         "babel-plugin-emotion": "^10.0.33",
+        "babel-plugin-const-enum": "^1.0.1",
```

Comment:
When I run locally, I get some yarn lock file updates, it might be worth running `yarn` again and pushing the lock changes